src/components/Board/Board.jsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import Task from "../TaskBox/Task";
+import './Board.css';
+import useBoardDrop from "../../hooks/useBoardDrop";
+
+const Board = ({ status, setTasks, todo, readyForDev, inProgress, done }) => {
+    const { dropRef, isOver } = useBoardDrop(status, setTasks);
+
+    let tasksToMap = todo;
```

Comment:
i would recommend to use useState for this

Reply:
done